src/main/java/pls_no_shinobu/videostore/model/User.java

```diff
@@ -144,14 +146,18 @@ public void setUsername(String username) throws IllegalArgumentException {
         this.username = username;
     }
 
-    protected String getPassword() {
```

Comment:
don't know why but this is red on my GitHub, and line 148 and line 154 too. But I think they are fine .-. Who am I to judge god himself.